hardhat.config.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-env node */
+require('dotenv').config()
+
+const mainnetFork = {
+  url: `https://mainnet.infura.io/v3/${process.env.REACT_APP_INFURA_KEY}`,
```

Comment:
confirming `REACT_APP_INFURA_KEY` is the correct key in the hardhat env

Reply:
yes -  hardhat needs to fork off of a mainnet archival node, and the infura JSON-RPC endpoint is used to access that archival node.